<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>850</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1200</width>
    <height>850</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1200</width>
    <height>850</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Classification Pills</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1200</width>
      <height>1000</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>1200</width>
      <height>1000</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>1200</width>
      <height>900</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true"> background-color: rgba(0, 0, 0, 0); /* Fully transparent background */</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tabPredict">
     <attribute name="title">
      <string/>
     </attribute>
     <widget class="QPushButton" name="btnBrowseImage">
      <property name="geometry">
       <rect>
        <x>960</x>
        <y>120</y>
        <width>201</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	border-radius: 5px;	
	color:rgb(255,255,255);
	background-color: rgb(27, 90, 225);
}
QPushButton:hover {
	border-radius: 5px;	
	background-color: rgb(136,189, 255);
}
QPushButton:pressed {	
	border-radius: 5px;	
	background-color: rgb(20, 180, 255);
}

QPushButton:disabled {	
	border-radius: 5px;	
	background-color: rgb(73, 131, 255);
}</string>
      </property>
      <property name="text">
       <string>Browse Images</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>10</y>
        <width>201</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 16pt &quot;Microsoft YaHei UI&quot; ;
 background-color: rgba(0, 0, 0, 0); /* Fully transparent background */
color: rgba(255, 255,255)</string>
      </property>
      <property name="text">
       <string>Steps</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnPredict">
      <property name="geometry">
       <rect>
        <x>960</x>
        <y>190</y>
        <width>201</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	border-radius: 5px;	
	color:rgb(255,255,255);
	background-color: rgb(27, 90, 225);
}
QPushButton:hover {
	border-radius: 5px;	
	background-color: rgb(136,189, 255);
}
QPushButton:pressed {	
	border-radius: 5px;	
	background-color: rgb(20, 180, 255);
}

QPushButton:disabled {	
	border-radius: 5px;	
	background-color: rgb(73, 131, 255);
}</string>
      </property>
      <property name="text">
       <string>Predict Class Of Pill</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>950</x>
        <y>100</y>
        <width>221</width>
        <height>161</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:rgb(242, 243, 244);
border:1px solid rgb(  214, 219, 223  );	
border-radius:5px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>930</x>
        <y>40</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Microsoft YaHei UI</family>
        <pointsize>16</pointsize>
        <weight>9</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 16pt &quot;Microsoft YaHei UI&quot; ;
background-color: rgba(0, 0, 0, 0); /* Fully transparent background */
color: rgba(255, 255,255)</string>
      </property>
      <property name="text">
       <string>Actions</string>
      </property>
     </widget>
     <widget class="QProgressBar" name="progressBar">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>90</y>
        <width>311</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"> QProgressBar::chunk {
	background-color: rgb(17, 132, 255)    ;
	 width: 20px;

 }</string>
      </property>
      <property name="value">
       <number>0</number>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="image_2">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1200</width>
        <height>1000</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>1200</width>
        <height>1000</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>1200</width>
        <height>1000</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../project test/imgLogo.jpg</pixmap>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="labelImage1">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>90</y>
        <width>221</width>
        <height>181</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:rgb(255, 255, 255);
border:1px solid rgb(  214, 219, 223  );	
border-radius:5px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="labelPredict1">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>220</y>
        <width>341</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background:rgb(255, 255, 255);
border:1px solid rgb(  214, 219, 223  );	
border-radius:5px;</string>
      </property>
      <property name="text">
       <string>Predict Type Of Disease : </string>
      </property>
     </widget>
     <widget class="QLabel" name="labelImage1_2">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>60</y>
        <width>221</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:rgb(255, 255, 255);
border:1px solid rgb(  214, 219, 223  );	
border-radius:5px;</string>
      </property>
      <property name="text">
       <string>Original Image</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_tr">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>340</y>
        <width>400</width>
        <height>380</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background:rgb(255, 255, 255);
border:1px solid rgb(  214, 219, 223  );	
border-radius:5px;</string>
      </property>
      <property name="lineWidth">
       <number>2</number>
      </property>
      <property name="text">
       <string>Text ......</string>
      </property>
      <property name="textFormat">
       <enum>Qt::PlainText</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
      <property name="margin">
       <number>5</number>
      </property>
      <property name="indent">
       <number>2</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>290</y>
        <width>151</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 16pt &quot;Microsoft YaHei UI&quot; ;
background-color: rgba(0, 0, 0, 0); /* Fully transparent background */
color: rgba(255, 255,255)</string>
      </property>
      <property name="text">
       <string>Description</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>290</y>
        <width>151</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 16pt &quot;Microsoft YaHei UI&quot; ;
background-color: rgba(0, 0, 0, 0); /* Fully transparent background */
color: rgba(255, 255,255)</string>
      </property>
      <property name="text">
       <string>Translation</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_des">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>340</y>
        <width>400</width>
        <height>380</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background:rgb(255, 255, 255);
border:1px solid rgb(  214, 219, 223  );	
border-radius:5px;</string>
      </property>
      <property name="lineWidth">
       <number>2</number>
      </property>
      <property name="text">
       <string>Text .......</string>
      </property>
      <property name="textFormat">
       <enum>Qt::PlainText</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
      <property name="margin">
       <number>5</number>
      </property>
      <property name="indent">
       <number>2</number>
      </property>
     </widget>
     <widget class="QLabel" name="labelPredict2">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>160</y>
        <width>341</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background:rgb(255, 255, 255);
border:1px solid rgb(  214, 219, 223  );	
border-radius:5px;</string>
      </property>
      <property name="text">
       <string>Predict Contain Of Image : </string>
      </property>
     </widget>
     <widget class="QLabel" name="label_new">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>90</y>
        <width>251</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background:rgb(255, 255, 255);
border:1px solid rgb(  214, 219, 223  );	
border-radius:5px;</string>
      </property>
      <property name="text">
       <string>Progress</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>790</x>
        <y>150</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: white;
border:1px solid rgb(  214, 219, 223  );
border-radius:5px;
padding: 5px;
font-size: 16px;
color: #333;

</string>
      </property>
     </widget>
     <zorder>image_2</zorder>
     <zorder>label_14</zorder>
     <zorder>label_10</zorder>
     <zorder>label_13</zorder>
     <zorder>btnBrowseImage</zorder>
     <zorder>btnPredict</zorder>
     <zorder>progressBar</zorder>
     <zorder>labelImage1</zorder>
     <zorder>labelPredict1</zorder>
     <zorder>labelImage1_2</zorder>
     <zorder>label_tr</zorder>
     <zorder>label_17</zorder>
     <zorder>label_18</zorder>
     <zorder>label_des</zorder>
     <zorder>labelPredict2</zorder>
     <zorder>label_new</zorder>
     <zorder>comboBox</zorder>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
